inst/project/utils.R

```diff
@@ -0,0 +1,244 @@
+`%||%` <- function(x, y) {
```

Comment:
Similar comment to the one about 'targets.R': I'd recommend reducing this file to the bare necessities.